packages/react/src/hooks/index.ts

```diff
@@ -24,11 +24,12 @@ type Logto = {
     | 'getOrganizationTokenClaims'
     | 'getIdToken'
     | 'getIdTokenClaims'
-    | 'signIn'
     | 'signOut'
     | 'fetchUserInfo'
   >
->;
+> &
+  // Manually pick the method with overloads since TypeScript cannot infer the correct type.
+  Pick<LogtoClient, 'signIn'>;
```

Comment:
Just curious why would this work? It doesn't seem to do anything different. 🤔

Reply:
idk, ts cannot infer via a remapping